desktop/core/src/desktop/js/ko/components/ko.jobBrowserLinks.js

```diff
@@ -32,15 +32,15 @@ const TEMPLATE = `
         <span>${I18n('Queries')}</span>
       </a>
     <!-- /ko -->
-    <!-- ko if: window.HAS_JOB_BROWSER -->
+    <!-- ko if: window.HAS_JOB_BROWSER && !window.getLastKnownConfig().has_computes -->
     <!-- ko ifnot: window.IS_K8S_ONLY -->
-      <a class="btn btn-flat" style="padding-right: 4px" title="${I18n(
+      <a class="btn btn-flat" id="jobs-browser" style="padding-right: 4px" title="${I18n(
         'Job browser'
       )}" data-bind="hueLink: '/jobbrowser#!jobs', click: function() { huePubSub.publish('hide.jobs.panel'); }">
         <span>${I18n(window.IS_MULTICLUSTER_ONLY ? 'Queries' : 'Jobs')}</span>
       </a>
     <!-- /ko -->
-    <button class="btn btn-flat btn-toggle-jobs-panel" title="${I18n(
+    <button class="btn btn-flat btn-toggle-jobs-panel" id="jobs-browser-button" title="${I18n(
```

Comment:
Do we need this change?

It seems like we're hiding the job browser buttons in the case where we have computes... And if that's so I don't think we'll need the introduction of the id params below the condition.